docs/release-notes.md

```diff
@@ -16,6 +16,14 @@ Packaging changes:
 
 
 
+## Afterburn 5.4.3 (2023-06-05)
+
+Minor changes:
+
+- Cargo update all dependencies to avoid missing data in vendor tarball
```

Comment:
The last time this happened, in 5.4.1, we said: `Fix missing static archives in vendor tarball`.  They're not static archives in this case; maybe `Fix incomplete package in vendor tarball`?

Re the `cargo update` part, normally dependency updates don't get a release note.  If we were correcting years of missed updates, I'd definitely say it was worth noting, but the last update was b3e25907f7935e6e01a057e8497dd370e1ab419b in January.  So I'd be inclined to skip.

Reply:
Sorry I half read the suggestion; no excuse I fixed it.